### PR TITLE
Refactored packet initialization to remove duplicate code.

### PR DESCRIPTION
**Pull request by dkolistratova**
Removed a bunch of duplicates (was noticed changing constants in pr #397 ) by the way, there was also a bug  packet.GetIPv6NoCheck().PayloadLen = SwapBytesUint16(uint16(UDPLen + plSize)) for icmp packet.

### CODE REVIEW

**Comment by ifilippov** [comment]
You should remove ethernet and ParseL3 from your functions
You should pass packet.GetIPv4NoCheck() to you function instead of packet

**Comment by dkolistratova** [comment]
@ifilippov 

then packet.Ether.EtherType = SwapBytesUint16(IPV6Number)
	packet.ParseL3()
lines will be duplicated everyrhere. 

Maybe call function "fillCommonDataForIPv6" and leave as is?

**Comment by ifilippov** [comment]
Why are you care about ParseL3 and don't care about ParseL4 which is also duplicated? I don't like an idea to move Parses anywhere. It is OK to duplicate them. Also I don't like an idea to move ethernet initialization anywhere because it is confusing.

File: packet/packet.go [hunk 1]
```diff
@@ -473,6 +473,18 @@ func InitEmptyPacket(packet *Packet, plSize uint) bool {
 	return true
 }
 
+func fillIPv4(packet *Packet, plLen uint16, nextProto uint8) {
```
Comment (ifilippov) [inline packet/packet.go]: Word default somewhere. fillIpv4Default for example.

**Comment by ifilippov** [comment]
Approved, you can merge.

**Pull request merged**